SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -230,11 +230,12 @@ SCDocHTMLRenderer {
 		// (Search.html, Browse.html, etc.) if necessary.
 		stream
 		<< "</title>\n"
+		<< "<link rel='stylesheet' href='" << baseDir << "/custom.css' type='text/css' />\n"
 		<< "<link rel='stylesheet' href='" << baseDir << "/scdoc.css' type='text/css' />\n"
 		<< "<link rel='stylesheet' href='" << baseDir << "/codemirror.css' type='text/css' />\n"
 		<< "<link rel='stylesheet' href='" << baseDir << "/editor.css' type='text/css' />\n"
 		<< "<link rel='stylesheet' href='" << baseDir << "/frontend.css' type='text/css' />\n"
-		<< "<link rel='stylesheet' href='" << baseDir << "/custom.css' type='text/css' />\n"
+    	<< "<link rel='stylesheet' id='scdoc-theme' href='" << baseDir << "/themes/default.css' type='text/css' />\n"
```

Comment:
Shouldn't custom.css be the last stylesheet loaded, after themes? I think it's supposed to be the last to be able to override everything

Reply:
Yes, thanks for the hint - changed it.